ios/Mattermost.xcodeproj/project.pbxproj

```diff
@@ -111,6 +110,18 @@
 		F006936FC2884C24A1321FC0 /* MaterialIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 356C9186FA374641A00EB2EA /* MaterialIcons.ttf */; };
 		F083DB472349411A8E6E7AAD /* OpenSans-LightItalic.ttf in Resources */ = {isa = PBXBuildFile; fileRef = BE17F630DB5D41FD93F32D22 /* OpenSans-LightItalic.ttf */; };
 		F23C99AA5FA10E457A76803A /* libPods-MattermostTests.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 4246DC09024BB33A3E491E25 /* libPods-MattermostTests.a */; };
+		DDE492F7425D451884DAA088 /* Roboto-Black.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 6EFF13DD24CE4E26953E598A /* Roboto-Black.ttf */; };
```

Comment:
Do we need to include these files on iOS even when we're not using the font?

Reply:
this happens automatically when doing `react-native link` I don't see how this hurts at all

Reply:
Yes we need the files

Reply:
Thank you for that list, I’m going to remove the font files and see if the font displays on Android ;)

Reply:
Ok offline decision is to leave the fonts as there might be some Android manufacturers who might not include the font in their devices.